Woodoku/src/entity/woodoku_board.py

```diff
@@ -1,10 +1,11 @@
-from typing import Dict, List, Tuple, Set, Iterable
+from typing import List, Tuple, Set, Iterable
 
 import numpy as np
 from numpy import ndarray
+
+from ui.utils import *
```

Comment:
```suggestion
from ui.utils import (ALL_BOLD_CROSS, BLOCK, BOLD_BOTTOM_JOIN, BOLD_HORIZONTAL,
                      BOLD_TOP_JOIN, BOLD_VERTICAL, BOTTOM_JOIN, BOTTOM_LEFT,
                      BOTTOM_RIGHT, CROSS, HORIZONTAL, HORIZONTAL_BOLD_CROSS,
                      LEFT_JOIN, RIGHT_JOIN, TOP_JOIN, TOP_LEFT, TOP_RIGHT,
                      VERTICAL_CROSS, black, green, inbetween, orange)
```
I don't know... just a suggestion